config/initializers/donor_app_host.rb

```diff
@@ -0,0 +1,2 @@
+host_names = YAML::load_file(File.join(Rails.root, 'config', 'donor_app_host.yml'))
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.